test/runtests.jl

```diff
@@ -1,23 +1,39 @@
 using NewPkgEval
 using Test
 
-v = NewPkgEval.build_julia("v1.2.0")
-
-pkgnames = ["JSON", "TimerOutputs", "Crayons", "Example"]
-pkgs = NewPkgEval.read_pkgs(pkgnames)
-results = NewPkgEval.run(pkgs, 2, v)
-for pkg in pkgnames
-    @test results[pkg] == :ok
-    output = read(joinpath(@__DIR__, "..", logs, "logs-"*string(v), pkg*".log"), String)
-    @test occursin("Testing $pkg tests passed", output)
+# determine the version to use
+const verstr = get(ENV, "JULIA_VERSION", string(VERSION))
+const v = try
+    ver = VersionNumber(verstr)
+    NewPkgEval.obtain_julia(ver)
+    ver
+catch
+    # either the string is an invalid version number (assume it is a git ref),
+    # or we couldn't obtain this version
+    NewPkgEval.build_julia(verstr)
```

Comment:
I wanted to CI the `build_julia` part as well. Does that still happen?

Reply:
Yeah, both env-var options are currently being built by Travis: https://github.com/JuliaComputing/NewPkgEval.jl/blob/e3ee6c04879642ba803cd8575d1189994ce4afed/.travis.yml#L10-L12

Reply:
Cool, thanks.